doc/api-crd-reference.md

```diff
@@ -326,7 +326,7 @@ spec:
 | **Field** | **json field**| **Type** | **Info** | **Required** |
 | --- | --- | --- | --- | --- |
 | Default | `default` | boolean | Sets the Plan as the default for developers to sign-up | Yes |
-| Approval Required | `aprovalRequired` | boolean | Defines if a final user requires approval from the admin to sign-up in a plan | Yes |
+| Approval Required | `approvalRequired` | boolean | Defines if a final user requires approval from the admin to sign-up in a plan | Yes |
```

Comment:
```suggestion
| Approval Required | `approvalRequired` | boolean | Defines if a final user requires approval from the admin to sign-up for a plan | Yes |
```

Reply:
- Sign up (two words) is a verb phrase.
- Sign-up (hyphenated) is a noun or an adjective.
- sign up for something

Reply:
thanks, @thomasmaas!